vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -30,14 +30,19 @@ module InlineParameterNameHints =
     let private doesFieldNameExist (field: FSharpField) = 
         not field.IsNameGenerated
 
-    let isMemberOrFunctionOrValueValidForHint (symbol: FSharpMemberOrFunctionOrValue) =
-        // is there a better way?
-        let isNotBuiltInOperator = 
-            symbol.DeclaringEntity 
-            |> Option.exists (fun entity -> entity.CompiledName <> "Operators")
-
-        symbol.IsFunction
-        && isNotBuiltInOperator // arguably, hints for those would be rather useless
+    let isMemberOrFunctionOrValueValidForHint (symbol: FSharpMemberOrFunctionOrValue) (symbolUse: FSharpSymbolUse) =
+        // make sure we're looking at a call site and not the definition
```

Comment:
IMO clear enough, feel free to remove the comment.

Reply:
I mean some tests for these things are the best but since it works rather as an optimization - it would be more useful to mention _that_ fact.